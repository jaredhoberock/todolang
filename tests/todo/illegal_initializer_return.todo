class Foo {
  fun init() {
    return "something else";
  }
}
