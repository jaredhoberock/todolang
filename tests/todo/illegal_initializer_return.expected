Runtime error: Can't return a value from an initializer.
